codalab/common.py

```diff
@@ -101,10 +101,12 @@ class State(object):
     RUNNING = 'running'   # Actually running
     READY = 'ready'       # Done running and succeeded
     FAILED = 'failed'     # Done running and failed
+    KILLED = 'killed'     # Killed by user
```

Comment:
I think having `killed` is fine, but the thing is that I think we've implicitly assumed that we had a closed set of states, and some parts of the code assume that (also the public stats worksheet).  So we should check those places, and generally have a plan for thinking about states - are we going to keep on increasing the number of states, or is this it?